app/cisc/cisc.go

```diff
@@ -290,8 +324,66 @@ func sshSync(c *cli.Context) error {
 	log.Fatal("Not yet implemented")
 	return nil
 }
-func proposeSendVoteUpdate(cfg *identity.Identity, p *identity.Config) {
-	log.ErrFatal(cfg.ProposeSend(p))
-	log.ErrFatal(cfg.ProposeVote(true))
-	log.ErrFatal(cfg.ConfigUpdate())
+
+func followAdd(c *cli.Context) error {
+	if c.NArg() < 2 {
+		log.Fatal("Please give a group-definition, an ID, and optionally a service-name of the skipchain to follow")
+	}
+	cfg, _ := loadConfig(c)
+	group := getGroup(c)
+	idBytes, err := hex.DecodeString(c.Args().Get(1))
+	log.ErrFatal(err)
+	id := identity.ID(idBytes)
+	newID, err := identity.NewIdentityFromCothority(group.Roster, id)
+	log.ErrFatal(err)
+	if c.NArg() == 3 {
+		newID.DeviceName = c.Args().Get(2)
+	} else {
+		var err error
+		newID.DeviceName, err = os.Hostname()
+		log.ErrFatal(err)
+	}
+	cfg.Follow = append(cfg.Follow, newID)
+	cfg.writeAuthorizedKeys(c)
+	// Identity needs to exist, else saving/loading will fail. For
+	// followers it doesn't matter if the identity will be overwritten,
+	// as it is not used.
+	cfg.Identity = newID
+	return cfg.saveConfig(c)
+}
+func followDel(c *cli.Context) error {
+	if c.NArg() != 1 {
+		log.Fatal("Please give id of skipchain to unfollow")
+	}
+	cfg := loadConfigOrFail(c)
+	idBytes, err := hex.DecodeString(c.Args().First())
+	log.ErrFatal(err)
+	idDel := identity.ID(idBytes)
+	var newSlice []*identity.Identity
+	for _, id := range cfg.Follow {
+		if !bytes.Equal(id.ID, idDel) {
+			newSlice = append(newSlice, id)
+		}
+	}
+	cfg.Follow = newSlice
+	cfg.writeAuthorizedKeys(c)
+	return cfg.saveConfig(c)
+}
+func followList(c *cli.Context) error {
+	cfg := loadConfigOrFail(c)
+	for _, id := range cfg.Follow {
+		log.Infof("Following ID: %x", id.ID)
+		log.Infof("As service %s getting ssh-keys from %s:",
+			id.DeviceName,
+			id.Config.GetIntKeys("ssh", id.DeviceName))
```

Comment:
1. it prints out all stored keys for all followers. I'll replace `As service` with `Follower`.
2. `Get.*Keys` handles `a:b:c`-keys with regard to the first, middle or last element. It's for handling the key/value 'database' of the identity-structure.
3. the `kvList` method (using argument `kv list`) does a more parser-friendly output. If you think it's worth it, I can do a JSON-compatible output in `kvList`. Or add a `--json` flag to the output-methods so that they print it JSON-compatible.